var/spack/repos/builtin/packages/camp/package.py

```diff
@@ -27,6 +27,11 @@ def cmake_args(self):
 
         options = []
 
+        # if blt is not in the spec, it means that there is no explicit blt
+        # dependency and an internal blt submodule is being used.
+        if 'blt' in spec:
```

Comment:
There's no code path that satisfies this conditional, you need to either `depends_on('blt', type='build')` or rely on blt being present as a submodule.

Reply:
Changed here and in chai to depend on blt explicitly.